components/Breadcrumb/src/breadcrumb.css

```diff
@@ -0,0 +1,125 @@
+.denhaag-breadcrumb {
+  --denhaag-breadcrumb-background-color: transparent;
+  color: var(--denhaag-breadcrumb-color);
+}
+
+.denhaag-breadcrumb__list {
+  block-size: 100%;
+  display: flex;
+}
+
+ol.denhaag-breadcrumb__list {
+  list-style: none;
+  margin-block-start: 0;
+  margin-block-end: 0;
+  padding-inline-start: 0;
+}
+
+.denhaag-breadcrumb__item,
+.denhaag-breadcrumb__link {
+  align-items: center;
+  display: flex;
+}
+
+.denhaag-breadcrumb__item:not(:first-child):not(:nth-last-child(2)):not(:last-child) {
+  --denhaag-breadcrumb-background-color: var(--denhaag-breadcrumb-dots-background-color, var(--denhaag-color-grey-1));
+}
+
+.denhaag-breadcrumb__item:not(:first-child):not(:nth-last-child(2)):not(:last-child):hover {
+  --denhaag-breadcrumb-background-color: var(
+    --denhaag-breadcrumb-dots-hover-background-color,
+    var(--denhaag-color-grey-2)
```

Comment:
Same as the comment above

Reply:
See my reaction in the topic https://github.com/nl-design-system/denhaag/pull/586#discussion_r764755985